tests/StreamTest.php

```diff
@@ -66,6 +66,15 @@ public function testChecksEof()
         $stream->close();
     }
 
+    public function testReadError()
+    {
+        $this->expectException(\RuntimeException::class);
+        $handle = fopen('php://temp', 'a');
```

Comment:
Well, that does work. You cannot read from the write-only stream. :)

Reply:
Yes it's seem work, but why the CI throw an error on BC Check ?